summoner-cli/summoner.cabal

```diff
@@ -19,6 +19,26 @@ extra-doc-files:     README.md
 tested-with:         GHC == 8.2.2
                    , GHC == 8.4.4
                    , GHC == 8.6.4
+extra-source-files:
+    test/golden/fullProject/*.yml
+    test/golden/fullProject/*.yaml
+    test/golden/fullProject/*.md
+    test/golden/fullProject/*.cabal
+    test/golden/fullProject/src/*.hs
+    test/golden/fullProject/app/*.hs
+    test/golden/fullProject/test/*.hs
+    test/golden/fullProject/benchmark/*.hs
+    test/golden/fullProject/.gitignore
+    test/golden/fullProject/.travis.yml
+    test/golden/fullProject/.stylish-haskell.yaml
+    test/golden/fullProject/stack-8.4.4.yaml
+    test/golden/fullProject/stack-8.2.2.yaml
+    test/golden/fullProject/stack-8.0.2.yaml
+    test/golden/fullProject/LICENSE
+
+    test/golden/smallProject/*.md
+    test/golden/smallProject/*.cabal
+    test/golden/smallProject/app/*.hs
```

Comment:
Heh :disappointed:

Reply:
I hope this part can be simplified once we move to `cabal-2.4`...